extension-readme.md

```diff
@@ -1,134 +1,97 @@
 # <a id="top"></a>AWS Toolkit
 
-The _AWS Toolkit_, or simply the _Toolkit_, is an extension that enables you to interact with certain services of [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/) from within the VS Code editor.
-
-The following screenshots show important parts of the Toolkit.
-
-## <a id="ui-components"></a>Fundamental UI Components
-
-### <a id="ui-components-aws-expl"></a>The AWS Explorer
-
-![Overview, AWS Explorer](./resources/marketplace/overview.png)
-
-### <a id="ui-components-cdk-expl"></a>The AWS CDK Explorer
-
-![Overview, AWS CDK Explorer](./resources/marketplace/overview-cdk.png)
-
-### <a id="open-command-palette"></a>AWS Commands in the Command Palette
-
-![Command Palette](./resources/marketplace/open-command-palette.gif)
-
----
+AWS Toolkit is an extension for Visual Studio Code that enables you to interact with [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/).
+See the [_AWS Toolkit for Visual Studio Code User Guide_](https://docs.aws.amazon.com/console/toolkit-for-vscode/welcome) for complete documentation.
 
 # <a id="contents"></a>Contents
 
--   [Feature Overview](#feature-overview)
--   [Setup](#additional-setup-steps)
 -   [Features](#features)
--   [Appendix A: AWS Commands](#aws-commands)
--   [Appendix B: Get Help](#get-help)
-
-{ [Return to Top](#top) }
-
----
-
-# <a id="feature-overview"></a>Feature Overview
-
-You can use the AWS Toolkit to interact with several AWS resources in various ways.
-
-These include the following:
-
--   AWS serverless applications
-
--   AWS Lambda functions
-
--   AWS CloudFormation stacks
-
--   AWS Cloud Development Kit (AWS CDK) applications
-
--   Amazon EventBridge schemas
-
--   Amazon Elastic Container Service (Amazon ECS) task definition files
+-   [Setup](#additional-setup-steps)
+-   [Appendix A: AWS commands](#aws-commands)
```

Comment:
Half of it is now appendix's, I think this fine as a section title without `Appendix`